### ISSUE TITLE
CVEs found in buildpack-dep

### ISSUE THREAD

**Issue by scriptmonkey** [docker-library/official-images#11923]
The CVEs have been fixed in the latest buildpack-dep.

The downstream images need to be rebuild, for our needs OpenJDK.  However, I would think that every image that relies on would need to be rebuilt.

If this is not the correct place to address this issue, where is the best place?

**Comment by scriptmonkey** [comment]
The CVEs in question are for libexpat1. I ran a scan against buildpack-deps:bullseye-curl, the FROM line in OpenJDK. When that scan came back clean, I assumed it was fixed in buildpack-deps:bullseye-curl.

It turns out that libexpat1 is not installed in buildpack-deps:bullseye-curl. libexpat1 gets installed as a dependency to fontconfig in the OpenJDK Dockerfile.

I think a rebuild of the bullseye OpenJDK would fix these CVEs.
